### PR TITLE
External config

### PR DESCRIPTION
**Pull request by luontola**
#613

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -104,80 +102,70 @@
...
-                {:output-dir "target/cljsbuild/public/js"
-                 :output-to "target/cljsbuild/public/js/app.js"
-                 :source-map "target/cljsbuild/public/js/app.js.map"
-                 :optimizations :advanced
-                 :pretty-print false
-                 :closure-warnings
-                 {:externs-validation :off :non-standard-jsdoc :off}
-                 :infer-externs :true ;; for window.rems.hooks to work
-                 :externs ["react/externs/react.js"]}}}}
+             :cljsbuild {:builds {:min {:source-paths ["src/cljc" "src/cljs"]
```
Comment (Koura) [inline project.clj]: +1 good stuff

File: src/clj/rems/auth/ldap.clj [hunk 2]
```diff
@@ -18,7 +18,8 @@
   [user password]
   (try
     (let [connection (assoc (getx-in env [:ldap :connection])
-                            :bind-dn user :password password)
+                       :bind-dn user
```
Comment (Koura) [inline src/clj/rems/auth/ldap.clj]: fix indentation

File: src/clj/rems/themes.clj [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+(defn load-default-theme []
+  (source/from-resource "theme-defaults.edn"))
+
+(defn load-theme [theme-path default-theme]
+  (merge default-theme
+         (when theme-path
+           (try
+             (assoc (source/from-file theme-path)
+               :theme-static-resources (.getPath (io/file (.getParentFile (io/file theme-path))
```
Comment (Koura) [inline src/clj/rems/themes.clj]: fix indentation

File: test/clj/rems/test/themes.clj [hunk 4]
```diff
@@ -1,40 +1,38 @@
...
-      (is (= default-theme
-             (themes/load-theme nil default-theme))))
-    (testing "non-existing theme gives default theme"
-      (is (= default-theme
-             (themes/load-theme "no-such-file.edn" default-theme))))
-    (testing "static resources can be placed in 'public' directory next to the theme file"
-      (is (= "lbr-theme/public"
-             (:theme-static-resources (themes/load-theme "lbr-theme/lbr.edn" default-theme)))))
-    (testing "overrides values from the default theme"
+  (testing "non-existing theme file is ignored silently"
```
Comment (Koura) [inline test/clj/rems/test/themes.clj]: why? The user experience is a lot better when there is actually a row in the log explaining why it was ignored.

Comment (luontola) [reply test/clj/rems/test/themes.clj]: How about just making it an error which prevents the application from starting? That would make detecting the problem even easier.

**Pull request merged**